Tasks/SshV0/ssh.ts

```diff
@@ -108,42 +104,55 @@ async function run() {
             if (runOptions === 'commands') {
                 //run commands specified by the user
                 for (const command of commands) {
-                    tl.debug('Running command ' + command + ' on remote machine.');
+                    tl.debug(`Running command ${command} on remote machine.`);
                     console.log(command);
                     const returnCode: string = await sshHelper.runCommandOnRemoteMachine(
                         command, sshClientConnection, remoteCmdOptions);
-                    tl.debug('Command ' + command + ' completed with return code = ' + returnCode);
+                    tl.debug(`Command ${command} completed with return code = ${returnCode}`);
                 }
-            } else { // both other runOptions: inline and script
-                //setup script path on remote machine relative to user's $HOME directory
-                const remoteScript = './' + path.basename(scriptFile);
-                let remoteScriptPath = '"' + remoteScript + '"';
-                const windowsEncodedRemoteScriptPath = remoteScriptPath;
-                const isWin = os.platform() === 'win32';
+            } else {
+                // both other runOptions: inline and script
+                // setup script path on remote machine relative to user's $HOME directory
+                const remoteScript: string = `./${path.basename(scriptFile)}`;
+                let remoteScriptPath: string = `"${remoteScript}"`;
+                const windowsEncodedRemoteScriptPath: string = remoteScriptPath;
+                const isWin: boolean = (os.platform() === 'win32');
                 if (isWin) {
-                    remoteScriptPath = '"' + remoteScript + "._unix" + '"';
+                    remoteScriptPath = `"${remoteScript}._unix"`;
+                }
+                tl.debug(`remoteScriptPath = ${remoteScriptPath}`);
+
+                //setup the scp configuration based on endpoint details
+                const scpConfig: sshHelper.ScpConfig = {
+                    host: hostname,
+                    port: port,
+                    username: username,
+                    path: remoteScript
+                };
+
+                if (privateKey && privateKey !== '') {
```

Comment:
```suggestion
                if (privateKey) {
```

Reply:
Thanks for the code suggestion! Fixed it.